orttraining/orttraining/python/training/utils/hooks/_zero_offload_subscriber.py

```diff
@@ -121,6 +168,23 @@ def configure_ort_compatible_zero_stage3(debug=False, stats_output_dir=None, sta
         raise RuntimeError("DeepSpeed is not installed, cannot configure ORT compatible ZeRO stage3.")
 
 
+try:
+    import deepspeed
+
+    def configure_ort_compatible_allgather_fn_zero_stage3():
```

Comment:
ci failed with `ImportError: cannot import name 'configure_ort_compatible_allgather_fn_zero_stage3' from 'onnxruntime.training.utils.hooks' (/home/onnxruntimedev/miniconda3/lib/python3.8/site-packages/onnxruntime/training/utils/hooks/__init__.py)` because `configure_ort_compatible_allgather_fn_zero_stage3` is not exposed in hooks module.

You can move combine the function body of `configure_ort_compatible_allgather_fn_zero_stage3` into existing `configure_ort_compatible_zero_stage3` I think.